src/databricks/labs/ucx/contexts/application.py

```diff
@@ -272,6 +277,27 @@ def table_ownership(self) -> TableOwnership:
             self.workspace_path_ownership,
         )
 
+    @cached_property
+    def default_owner_group(self) -> str | None:
+        owner_group = None
+        if self.config.default_owner_group:
+            if not self.group_manager.validate_owner_group(self.config.default_owner_group):
+                logger.warning("Default owner group is not valid, falling back to administrator ownership.")
```

Comment:
`ApplicationContext` must only have object initialisation logic. this has to happen after constructor of `DefaultSecurableOwnership` is initialized, in one of the methods.

**`ApplicationContext` must not make calls to API.**